### PR TITLE
Fresh finnish translations

### PR DESCRIPTION
**Pull request by olmari**
Signed-off-by: Sami Olmari <sami@olmari.fi>

### CODE REVIEW

File: emails/locale/fi/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -3,76 +3,80 @@
...
-msgstr ""
+msgstr "Sisältö"
 
 #: emails/models.py:36
 msgid ""
 "Content of the email. All emails will start with default 'View this message "
 "in browser' and end with 'You are receiving this message because') texts"
 msgstr ""
+"Sähköpostin sisältö. Kaikkien viestien pitäisi alkaa oletus 'Katsele "
+"viestiä selaimessa' ja loppua 'Sait tämän viestin koska' teksteillä"
```
Comment (tswfi) [inline emails/locale/fi/LC_MESSAGES/django.po:55]:
```suggestion
"Sähköpostin sisältö. Kaikki viestit alkavat 'Katsele "
"viestiä selaimessa' ja loppuvat 'Sait tämän viestin koska' teksteillä"
```

Tosin tälle ei taida olla test caseja :D

File: users/locale/fi/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -7,8 +7,8 @@ msgid ""
 msgstr ""
 "Project-Id-Version: \n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2020-07-16 14:08+0000\n"
-"PO-Revision-Date: 2020-07-16 17:09+0300\n"
+"POT-Creation-Date: 2020-09-26 13:25+0300\n"
+"PO-Revision-Date: 2020-09-26 13:40+0300\n"
 "Last-Translator: Tatu Wikman <tatu.wikman@gmail.com>\n"
```
Comment (tswfi) [inline users/locale/fi/LC_MESSAGES/django.po:12]: Might want to change these when updating translations :)

File: www/locale/fi/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -417,11 +419,11 @@ msgstr ""
...
 #, python-format
 msgid ""
-"Access to space should be paid every month with reference number %(ref)s\n"
-"to bank account %(bank_iban)s. Service price is %(cost)s."
+"This service can be paid with reference number %(ref)s\n"
+"                to bank account %(bank_iban)s. Service price is %(cost)s."
 msgstr ""
-"Tilankäyttöoikeus tulee maksaa kuukausittain viitenumerolla %(ref)s tilille %(bank_iban)s. "
-"Palvelun hinta on %(cost)s."
+"Tilankäyttöoikeus tulee maksaa kuukausittain viitenumerolla %(ref)s\n"
```
Comment (tswfi) [inline www/locale/fi/LC_MESSAGES/django.po:425]:
```suggestion
"Tämä palvelu tulee maksaa kuukausittain viitenumerolla %(ref)s\n"
```

The service can be something different thank tilankäyttöoikeus.

Comment (tswfi) [reply www/locale/fi/LC_MESSAGES/django.po:425]: hmm, even "palvelu" is a bit strange here.. maybe this should contain the service name?

Comment (olmari) [reply www/locale/fi/LC_MESSAGES/django.po:425]: Maybe, but needs also mod to english, so let's do that separately :)

File: users/locale/fi/LC_MESSAGES/django.po (hunk 2 above)
Comment (olmari) [reply users/locale/fi/LC_MESSAGES/django.po:12]: So... POedit software isn't perfect either it seems if it doens't touch those on save... meh.. :)

**Pull request merged**